### ISSUE TITLE
[Feature Request] Command Line Option: Skip Database Scan / Force Database Scan

### ISSUE THREAD

**Issue by cRaZy-bisCuiT** [UltraStar-Deluxe/USDX#322]
### Actual behaviour
UltrastarDeluxe scans the SongDir to build a database of Songs including covers, lyrics etc. The first launch of the program may be very long. If you have a large library (e.g. > 8k songs) it could take more then 10 minutes for the game to launch

### Expected behaviour

I'd like to have two command line options

-noscan -> the database will be untouched, no scan will happen. This is usefull when you did not change any songs/covers/etc. in your SongDir
-forcescan -> this will completely clear the database and create a new one. As a debug option.

No command line option should reflect the current behavior.

### Steps to reproduce

Launch the game. Especially with a large library.

### Details

Provide some additional information:

- USDX version: 2017.8.0 dev Build (as of 2017_11_13)
- Operating System + version: Arch Linux 4.13 + Mesa 17.2.5

**Comment by dgruss** [comment]
it is not necessary to skip the entire song scan.
#1121 parses all files, all headers, just skips the notes parsing part. and that speeds up loading time to just a few seconds then.

**Comment by dgruss** [comment]
this can be closed with #1121 merged

**Comment by barbeque-squared** [comment]
Yeah I would agree startup is a lot faster now. I'm also pretty sure this feature request would completely break if you ever delete or rename or (much more common) temporarily disable part of your library (eg my library is split into two parts, and it definitely keeps the scores of things that I temporarily disable, but it wouldn't know that if I did the proposed `-noscan`)

**Comment by dgruss** [comment]
i agree, this is in line with what OP intended
